Nighttime-Operations/Auxiliary-Telescope/AT-On-Sky/CWFS.rst

```diff
@@ -4,11 +4,92 @@
 
 .. _AT-On-sky-CWFS:
 
-#######################
-CWFS
-#######################
+#################################
+Focus, center and absorb offsets
+#################################
+
+.. _Beginning_of_the_night-the-Telescope-Overview:
+
+Overview
+========
+
+This procedure must be followed by the observers before the beginning of dark time, prior to starting observations. 
+It will focus the telescope, center a target and absorb the pointing offsets. 
+
+.. _Beginning_of_the_night-Prerequisites:
+
+Prerequisites
+=============
+- The telescope is ready for on-sky operations: it has been vented (if allowed), mirror cover and dome shutter are open, AOS corrections are enabled. (:ref:`Prepare for on-sky <AT-Pre-Open-for-On-Sky-Operations>`)
+- The sky brightness is low enough that a CWFS can converge succesfully. This condition is normally achieved around 10-15 minutes before the end of nautical twilight or dark time. 
```

Comment:
The "This condition..." sentence should be on a new line to use semantic line breaks.

Reply:
It appeared in a different bullet point, but I understand how to fix it now. 👍